10_rest/willshen8/pkg/puppy/storer_test.go

```diff
@@ -0,0 +1,121 @@
+package puppy
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+var (
+	out       io.Writer = os.Stdout
+	ReadError           = Error{Code: NonExistentPuppy,
```

Comment:
this looks weird as a code formating. Can you remove those spaces and rerun`go fmt`?

Reply:
My editor is set to auto-go-fmt. I ran the command manually as well. Linter would have picked up if it didn't like that.